tutoraspects/templates/aspects/apps/superset/pythonpath/create_assets.py

```diff
@@ -227,9 +227,15 @@ def generate_translated_dashboard_filters(copy, language):
     metadata = copy.get("metadata", {})
 
     for filter in metadata.get("native_filter_configuration", []):
-        translation = get_translation(filter["name"], language)
-
-        filter["name"] = translation
+        if "name" in filter:
+            translation = get_translation(filter["name"], language)
+            filter["name"] = translation
+        elif "description" in filter:
```

Comment:
What if a filter has both? Can you separate this?

Reply:
All set, it will now translate either if they exist and not error if those keys don't exist.